src/plugins/intel_cpu/src/cpu_map_scheduling.cpp

```diff
@@ -70,33 +71,37 @@ std::vector<std::vector<int>> apply_hyper_threading(bool& input_ht_hint,
 }
 
 bool get_cpu_pinning(bool& input_value,
-                     const bool input_changed,
-                     const int num_streams,
-                     const Config::LatencyThreadingMode latency_threading_mode,
-                     const std::vector<std::vector<int>>& proc_type_table) {
-    int result_value;
-    int num_sockets = get_default_latency_streams(latency_threading_mode);
-    bool latency = num_streams <= num_sockets && num_streams > 0;
+                     const bool& input_changed,
+                     const std::vector<std::vector<int>>& streams_info_table) {
+    bool result_value;
 
+#if defined(__APPLE__)
+    result_value = false;
+#else
     if (input_changed) {
         result_value = input_value;
     } else {
-        result_value = true;
-        if (proc_type_table[0][EFFICIENT_CORE_PROC] > 0 &&
-            proc_type_table[0][EFFICIENT_CORE_PROC] < proc_type_table[0][ALL_PROC]) {
-            result_value = latency ? false : true;
-        }
-    }
-#if (OV_THREAD == OV_THREAD_TBB || OV_THREAD == OV_THREAD_TBB_AUTO)
 #    if defined(_WIN32)
-    if (proc_type_table.size() > 1) {
         result_value = false;
```

Comment:
Does it mean that cpu binding is not supported in windows ?  Is it not support if user set `enableCpuPinning=true` ?